### DISCUSSION TITLE
Can't seem to get the Swell categories in Algolia

### DISCUSSION THREAD

**Discussion by christiancode14** [swellstores/community#153] [Help] [answered]
Hi y'all,

I'm trying to sync all product and category data to Algolia using the official Algolia integration. But unfortunately I can't seem to get the categories showing up in Algolia. Is it even possible to generate an Categories index in Algolia?

**Comment by sags95** [comment, marked as answer]
@christiancode14 With the Algolia integration only product records are synced, but you should see the `categories` field in Algolia. 
<img width="880" alt="Screenshot 2023-02-28 at 10 11 43 AM" src="https://user-images.githubusercontent.com/9522171/221927513-0c0973fb-1682-436d-b8ac-e619daa848fb.png">
